### DISCUSSION TITLE
Unwanted downloads in my queue

### DISCUSSION THREAD

**Discussion by bambury** [nicotine-plus/nicotine-plus#2126] [Q&A] [answered]
Sometimes I find unwanted downloads in my queue. I'm positive I didn't start them, it's music I don't know at all from users I don't know at all.

I didn't open an issue for this because I honestly have no idea whether it's Nicotine's fault, but it's quite annoying.
Is there a feature I don't know about that allows users to actively send files to other users, without their consent? I doubt it, but it never hurts to ask, especially since these downloads end up in the "Save buddies' uploads to:" folder, which is separate from the default one.
Again, the users I get these downloads from are not in my buddies list.

I'll try and collect detail information and logging when this happens again.

Thanks for any replies or insights!

**Comment by mathiascode** [comment, marked as answer]
The old Soulseek NS client has a feature to send random files to users it downloads from. If you send a few usernames to my email (on my profile), as well as some debug logs (right-click the log pane -> log categories -> enable connections, messages, transfers and downloads), I could check which client they're using.

I believe you might have changed an option to allow uploads from everyone in Preferences -> Downloads. I suggest changing it back to "buddies" or "no one".

**Reply by bambury** [reply]
Ah, so that's it! At this point I don't think you should even bother to look into this any further. I'll just change that setting. Thank you! :)

**Reply by mathiascode** [reply]
No problem. If you browse their files, they're very likely to use Soulseek NS if the folder paths are lowercase and start with a drive letter (e.g. c:\).
